test/WebJobs.Script.Tests/MetricsEventManagerTests.cs

```diff
@@ -72,20 +72,6 @@ public class MetricsEventManagerTests
                 string.Format("There are events with different execution id. List:{0} Invalid entries:{1}",
                     SerializeFunctionExecutionEventArguments(argsList),
                     SerializeFunctionExecutionEventArguments(invalidArgsList)));
-
```

Comment:
Why removing all this? It doesn't seem that your changes should have affected this test?